lib/bucky/core/test_core/test_case_loader.rb

```diff
@@ -148,6 +148,12 @@ def refine_case_with_option(test_cond, suite)
               suite[:cases].delete_if { |c| c[:labels].nil? }
               suite[:cases].delete_if { |c| !(test_cond[:label].sort - [c[:labels]].flatten.sort).empty? }
             end
+            # Exclude by label
+            if test_cond.key? :xlabel
+              # Delete test case that have specify label
+              suite[:cases].delete_if { |c| c[:labels].nil? }
+              suite[:cases].delete_if { |c| (test_cond[:xlabel].sort - [c[:labels]].flatten.sort).empty? }
```

Comment:
```suggestion
              suite[:cases].delete_if { |c| (test_cond[:xlabel].sort - [c[:labels]].flatten.sort).length < test_cond[:xlabel].length }
```

Reply:
I think we should discuss the specification of `exclude option`.
The following are my ideas.
PLZ tell me yours.
```
labels: # suite label
  - critical
cases:
  - case_name: test_1
    labels:
    - top
  - case_name: test_2
    labels:
    - detail
```

bucky run --xl critical
-> No test run.
bucky run --xl top,detail
-> No test run.
bucky run --xl top
-> test_2 run.
bucky run --xl critical,top
-> test_2 run.

Reply:
Same as the l(label) option, the every given parameter in option will use the AND logic at first,
then exclude from the test cases.

In this case :
```
labels: # suite label
  - critical
cases:
  - case_name: test_1
    labels:
    - top
  - case_name: test_2
    labels:
    - detail
```

```
bucky run --xl critical
Exclude the case with critical label
-> No test run.
```
```
bucky run --xl top,detail
Exclude the case with top and also with detail
-> test_1, test_2 run
```
```
bucky run --xl top
Exclude the case with top label
-> test_2 run.
```
```
bucky run --xl critical,top
Exclude the case with critical label and also with top label
-> test_2 run.
```

Reply:
The idea is same as --label, it will AND every input first then start to specific the test case.
--exclude_label also AND every input first then start to specific the test case.

Reply:
I got it! TY for explaining.